app/views/va_common/_header.html.erb

```diff
@@ -53,51 +52,35 @@
   <!-- /header alert box -->
   <div class="row full">
     <div class="small-6 medium-4 columns">
-      <h1 role="banner"><a href="/">Vets.gov</a></h1>
+      <h1 role="banner" class="va-header-logo"><a href="/">Vets.gov</a></h1>
     </div>
 
     <!-- Mobile search button -->
-    <div class="hide-for-medium-up search-button" id="trigger-overlay-button">
-      <button type="button" data-reveal-id="mobile-menu" class="usa-button usa-button-outline-inverse" aria-hidden="true">Search</button>
-    </div>
-
-    <div id="mobile-menu" class="reveal-modal overlay overlay-fullscreen" data-reveal aria-labelledby="modalTitle" aria-hidden="true" role="dialog" data-options="close_on_background_click:false">
-      <button type="usa-button" id="overlay-close-button" class="overlay-close close-reveal-modal" aria-label="Close">Close</button>
-
-      <div class="menu">
-      <form accept-charset="UTF-8" action="https://search.vets.gov/search" id="search_form" method="get">
-        <div style="margin:0;padding:0;display:inline">
-          <input name="utf8" type="hidden" value="&#x2713;" /></div>
-          <input id="affiliate" name="affiliate" type="hidden" value="vets.gov_search" />
-          <label for="mobile-query">Search:</label>
-          <div class="row collapse">
-            <div class="small-9 columns">
-              <input autocomplete="off" class="usagov-search-autocomplete" id="mobile-query" name="query" type="text" />
-            </div>
-            <div class="small-3 columns">
-              <input name="commit" type="submit" value="Search" />
-            </div>
-          </div>
-        </form>
-      </div>
```

Comment:
Yes. At one point there was a mobile search box and a desktop search box. They've since -- as far as I can tell -- been merged into one search box that's displayed for both (albeit differently). This is why I want to test in a project, juuuuust in case I'm wrong.